cards.js

```diff
@@ -342,11 +342,10 @@ DECK_DEFINITONS =
             , [true,  "21", "* %move% +0", "* %attack% +0", "** %pull% 1", "* %air%"]
             , [true,  "21", "* %move% +0", "* %attack% +0", "** %pull% 1", "* %air%"]
             , [false, "29", "* %move% +0", "* %attack% -1", "** %target% 2", "** %air%%use_element%: %push% 2"]
-            , [false, "37", "* %move% +0", "* %attack% +0 <div style='display: inline-block; width: 0; margin-top: -1.5em; vertical-align: bottom'>%aoe-4-with-black%</div>", "** %air%%use_element%: +1 %attack% <div style='display: inline-block; margin-right: -3em'>%aoe-circle-with-side-black%</div>"]
+            , [false, "37", "* %move% +0", "* %attack% +0 <div style='display: inline-block; width: 0; margin-top: -1.5em; vertical-align: bottom'>%aoe-4-with-black%</div>", "** %air%%use_element%: %attack% +1 <div style='display: inline-block; margin-right: -3em'>%aoe-circle-with-side-black%</div>"]

```

Comment:
My understanding is that if the demon can consume air, then you add one to the attack value and expand the AOE, but there's still only one attack action.

Regardless of rule interpretation, IMO we should match the lower portion of what is written on this card. The fact that the upper Move/Attack typos were noted in the FAQ but nothing about the lower portion was mentioned strongly implies to me that +1 Attack is the correct order.